packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -4,16 +4,17 @@
 
 import 'dart:html' as html;
 
+import 'package:devtools/src/ui/fake_flutter/dart_ui/dart_ui.dart';
+import 'package:devtools/src/ui/material_icons.dart';
```

Comment:
We should use relative imports here -

Reply:
can't wait till the quick fix automatically creates relative urls. Before then it could be nice to also have a lint that warns about using an absolute import when a relative one is possible. @pq